tests/sentry/web/api/tests.py

```diff
@@ -416,7 +416,7 @@ def test_scrubs_ip_address(self, mock_insert_data_to_database):
         resp = self._postWithHeader(body)
         assert resp.status_code == 200, (resp.status_code, resp.content)
 
-        call_data = mock_insert_data_to_database.call_args[0][0]
+        call_data = mock_insert_data_to_database.call_args[0][1]
```

Comment:
is there a way to get mock call_args by name? so we don't have to assume the index of the arg like this?

Reply:
No :( the first `[0]` is `*args` and so it's a tuple of nameless call arguments. We'd have to actually call `insert_data_to_database` with kwargs if we wanted to check them by name.